IFC4x3/Properties/c/CatenaryWireType_33lc5OXGT9FexvS$tyrx03/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CatenaryWireType_33lc5OXGT9FexvStyrx03" Name="CatenaryWireType" UniqueId="c3be6158-8507-493e-8ef9-73fdfcd7b003" PrimaryDataType="IfcLabel" />
```

Comment:
Would this not be better modeled in `IfcCableSegment`?

Reply:
No, even if the catenary is carrying current, the main function is to support the contact wire.